src/towncrier/test/test_build.py

```diff
@@ -70,9 +78,13 @@ def _test_command(self, command):
                 --------
 
                 - Baz levitation (baz)
-                - Baz fix levitation (#2)
+                - Baz fix levitation (fix-1.2)
```

Comment:
I think that this need a better comment.

I don't really understant what is the use case here.

This is the bad part of bundling all kind of scenarios in a single test case.

```
            # Towncrier supports files that have a dot in the name of the
            # newsfragment
            with open("foo/newsfragments/fix-1.2.feature", "w") as f:
                f.write("Baz fix levitation")
```

since it has no orphan marker, I would consider it an error, and the file should be rejected.

Reply:
As per my comment above, I think this snippet was parsed wrongly. But I cannot find anything in the docs about it the only reference is the comment (_builder.py):
```
# NOTE: This allows news fragment names like fix-1.2.3.feature or
# something-cool.feature.ext for projects that don't use ticket
# numbers in news fragment names.
```

And I do not think that in that case "3" was meant to be an issue number.

Reply:
Also I read the comment you cited as the "name of the newsfragment" being `fix-1.2`.